### PR TITLE
Code quality: Javadoc fixes for main and test command classes.

### PR DESCRIPTION
**Pull request by reddevil1313**

### CODE REVIEW

File: src/test/java/seedu/programmer/logic/commands/AddLabCommandTest.java [hunk 1]
```diff
@@ -18,6 +18,10 @@
 import seedu.programmer.model.student.Lab;
 import seedu.programmer.testutil.LabBuilder;
 
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
```
Comment (ZhaoPeiduo) [inline src/test/java/seedu/programmer/logic/commands/AddLabCommandTest.java:22]: I think the integration tests are done in a separate test class called `AddCommandIntegrationTest`!

Comment (reddevil1313) [reply src/test/java/seedu/programmer/logic/commands/AddLabCommandTest.java:22]: okay, makes sense!

Comment (reddevil1313) [reply src/test/java/seedu/programmer/logic/commands/AddLabCommandTest.java:22]: wait, I think you are confusing this file with AddCommandTest.java

Comment (ZhaoPeiduo) [reply src/test/java/seedu/programmer/logic/commands/AddLabCommandTest.java:22]: Yep, sorry abt that

**Review by ZhaoPeiduo — APPROVED** [review]
LGTM

**Pull request merged**